README.md

```diff
@@ -1 +1,51 @@
 # Logz.io Logs for Salesforce Commerce Cloud
+
+A Docker image purpose-built to monitor Salesforce Commerce Cloud (fka Demandware) logs
+
+## Usage
+
+### 1. Pull the Docker image
+
+Use the following docker command to run your container
+
+Donwload the logzio/webdav-fetcher image:
+
+```sh
+docker pull logzio/webdav-fetcher:latest
+```
+
+### 2. Run the container
+
+Use the following docker command to run your container.
+
+```sh
+docker run -d -e LOGZIO_SHIPPING_KEY=<logzio_shipping_token>  \
+-e SFCC_HOSTNAME=<your_sfcc_host> \
+-e SFCC_CLIENT_ID=<your_sfcc_client_id> \
+-e SFCC_CLIENT_SECRET=<your_sfcc_client_secret> \
+-e LOGZIO_LISTENER_URL=<logzio_listener_url> logzio/webdav-fetcher:latest
+```
+
+| Parameter           | Description                                                                                                                                                                                                                                                                                                                                   | Required |
+| ------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------: |
+| LOGZIO_LISTENER_URL | Determines protocol, listener host, and port. For example, https://<<LOGZIO_LISTENER_URL>>:8071.Replace <<LOGZIO_LISTENER_URL>> with your region's listener host (for example, listener.logz.io). For more information on finding your account's region, see [Account region](https://docs.logz.io/user-guide/accounts/account-region.html) . |      Yes |
+| LOGZIO_SHIPPING_KEY | The [token](https://app.logz.io/#/dashboard/settings/general) of the account you want to ship to.                                                                                                                                                                                                                                             |      Yes |
+| SFCC_HOSTNAME       | Hostname from what host need to send logs                                                                                                                                                                                                                                                                                                     |      Yes |
```

Comment:
Can you give an example on how it should looks like?

Reply:
<img width="839" alt="Screen Shot 2022-01-24 at 11 35 26" src="https://user-images.githubusercontent.com/11467909/150758637-b05af98d-e660-42af-8797-735d10f32786.png">

Reply:
@resdenia  - example in the doc itself - not for me , for the customer who will use this integration
How SFCC_hostname should look like